kong/ids.go

```diff
@@ -0,0 +1,68 @@
+package kong
+
+import (
+	"fmt"
+
+	"github.com/gofrs/uuid"
```

Comment:
Why using a new dependency,  when `github.com/google/uuid` is already used and offer the possibility to create Uuidv5

Reply:
Good point, fixed that.